inst/crunch-test.R

```diff
@@ -9,7 +9,21 @@ skip_locally <- function (...) {
 }
 
 loadCube <- function (filename) {
-    crunch:::CrunchCube(jsonlite::fromJSON(filename, simplifyVector=FALSE)$value)
+    file <- system.file(filename, package = "crunch")
+    if (nchar(file) > 0) {
+        filename <- file
+    }
+
+    cube_json <- jsonlite::fromJSON(filename, simplifyVector=FALSE)
+
```

Comment:
Why not `if ("value" %in% names(cube_json)) cube_json <- cube_json$value`? `tryCatch` makes it seem like we don't know the shape of data we're getting.